### PR TITLE
chore: added elasticsearch scrapers

### PR DESCRIPTION
**Pull request by Anyitechs**
This PR uses ticket #16 as a guide to add elasticsearch scrapers, but this is achieved by creating duplicate files for the scrapers rather than making changes to the app search scraper.

### CODE REVIEW

**Comment by Anyitechs** [comment]
@adamjonas please kindly review.

File: common/util-scraper.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    console.log("Done");
+}
+
+module.exports = {
+    create_batches,
+    index_documents,
+    fetch_with_retry
+};
```
Comment (adamjonas) [inline common/util-scraper.js]: EOF newline needed

File: bitcoin.stackexchange.com/main-scraper.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    index=os.getenv("INDEX"),
+                    actions=docs[i:i+100]
+                )
+                success = True
+            except:
+                import time
+                time.sleep(10)
+                pass
+    
+        i += 100
```
Comment (adamjonas) [inline bitcoin.stackexchange.com/main-scraper.py]: EOF newline

**Review by adamjonas — COMMENTED** [review]
based on the file name this is hard to differentiate this work for the elastic search scrapers from the app search scrapers. Can we put it their own dirs or something to make it clearer?

**Comment by ivssh** [comment]
Looks good to me :). Have we tested this by deploying it anywhere?

**Comment by adamjonas** [comment]
The only place this is deployed is with GitHub actions. See #21

**Pull request merged**